src/database/models/accountEstimationModel.js

```diff
@@ -11,14 +11,33 @@ module.exports = (db) => { return db.define("estimations", {
   totalNetFlowRate: {
     type: Sequelize.STRING
   },
-  totalBalance: {
+  availableBalance: {
     type: Sequelize.STRING
   },
-  zestimation: {
+  totalCFADeposit: {
+    type: Sequelize.STRING
+  },
+  estimation: {
```

Comment:
Is because of this context dependancy that I didn't rename it. I could rename but would also need to create a placeholder for all three estimation output.

Reply:
I see. Feels like code smell to me, thus I added it to https://github.com/superfluid-finance/superfluid-sentinel/issues/123